
Разрешаем конфликт коммитов

git push origin testbranch ->

 ```       To github.com:PyPro2022/Test_repository.git
    ! [rejected]        testbranch -> testbranch (fetch first)
    error: не удалось отправить некоторые ссылки в «git@github.com:PyPro2022/Test_repository.git»
    подсказка: Обновления были отклонены, так как внешний репозиторий содержит
    подсказка: изменения, которых у вас нет в вашем локальном репозитории.
    подсказка: Обычно, это связанно с тем, что кто-то уже отправил изменения в
    подсказка: то же место. Перед повторной отправкой ваших изменений, вам нужно
    подсказка: забрать и слить изменения из внешнего репозитория себе
    подсказка: (например, с помощью «git pull …»).
    подсказка: Для дополнительной информации, смотрите «Note about fast-forwards»
    подсказка: в «git push --help».```

 1. Делаем git pull origin testbranch
 2. Получаем файл с конфликтом

```
        <<<<<<< HEAD
        ghvhgvhjgvkhjvlhjbhj  444444 pycharm

        =======
        ghvhgvhjgvkhjvlhjbhj rep
        33333
        >>>>>>> 17fc7f759870feed5a8ea83b2181167ed20a160a
```

3. Ручками разрешаем конфликт
4. Делаем git add -> commit -m 'massage'
5. Делаем git push origin testbranch
